rasa/importers/multi_project.py

```diff
@@ -162,13 +164,39 @@ def _is_in_imported_paths(self, path) -> bool:
 
     def add_import(self, path: Text) -> None:
         self._imports.append(path)
-
+    
+    def _log_merge_infos(self, domains) -> None:
```

Comment:
please add unit tests for this

Reply:
I would be happy if you could give me a hand on this one (not sure what kind of unit test you have in mind + I did not dig into your unit test ways of working)!